ui/tests/helpers/sync/sync-selectors.js

```diff
@@ -85,7 +85,7 @@ export const PAGE = {
   saveButton: '[data-test-save]',
   toolbar: (btnText) => `[data-test-toolbar="${btnText}"]`,
   form: {
-    enableInput: (attr) => `[data-test-enable-field="${attr}"] [data-test-icon="edit"]`,
+    enableInput: (attr) => `[data-test-enable-field="${attr}"] [data-test-icon="edit"]`, // TODO duplicated in general-selectors as this component became more widely used
```

Comment:
Should this TODO be handled now, or future?

Reply:
future. it's a lot of file changes.